lib/auth/windows/ldap.go

```diff
@@ -100,13 +100,17 @@ const searchPageSize = 1000
 // is closed. Callers should check for trace.ConnectionProblem errors
 // and provide a new client with [SetClient].
 type LDAPClient struct {
-	// Cfg is the LDAPConfig
-	Cfg LDAPConfig
-
```

Comment:
Thanks for this cleanup here.

Reply:
I wanted to lower the config surface to make `LDAPClient` easier to use.